pair-filter/pair_filter/image_chooser.py

```diff
@@ -134,8 +134,22 @@ def get_image_tag(self) -> Optional[str]:
         # Does not match 'travis-ci-garnet-trusty-1503972833'
         match_obj_tag = re.search(r'(travis-ci(-[a-z]+)+-[0-9]+-[0-9a-z]+)', instance)
         if not match_obj_tag:
+            # Matches 'travisci/ci-garnet:packer-1503972846'
+            found_match_obj_tag = re.search(r'(travisci/ci-([a-z]+):([a-z]+)-[0-9a-z]+)', instance)
+            if not found_match_obj_tag:
+                return None
+            instance_tag = found_match_obj_tag.group(1)
+            instance_tag = instance_tag.split(':')
+            repo_name = instance_tag[0]
+            images = self.dockerhub_images.get(repo_name, '')
+            if not images:
+                return None
+            for image in images:
+                if instance_tag[1] == image:
```

Comment:
Since we are save `instance_tag[0]` to `repo` could we save `instance_tag[1]` to a variable like `image_tag`?